### PR TITLE
Use full length screenshot for live demo image.

### PR DESCRIPTION
**Pull request by budzanowski**
This PR enables usage of full length screenshots in Theme Sheet.
To enable this change wpcom api v1.2 was updated and it now has
screenshots array with images applicable for Theme Sheet. First image
in this array is a full length theme screenshot.

Currently not all Themes have full length sceenshot in place so you have to look
for example on themes like `babylog` to see this PR in action.

Before:

After:

 

Test live: https://calypso.live/theme/babylog?branch=update/use-long-screenshot-for-theme-sheet-img

### CODE REVIEW

File: client/my-sites/theme/main.jsx [hunk 1]
```diff
@@ -125,8 +126,16 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
+	getFullLengthScreenshot() {
+		/**
+		 * Location of full length theme screenshot in screenshots array
+		 * depends on themes API v1.2 screenshots field
+		 */
+		return this.props.screenshots[ 0 ];
```
Comment (seear) [inline client/my-sites/theme/main.jsx]: This array access causes problems for me before the theme data is loaded.

Testcase
* new tab
* go to /themes
* click on any theme

**Comment by folletto** [comment]
Design wise: 👍  

---

When navigating back/forth from the showcase to the sheet, I get for some themes:
 
screen shot 2016-07-25 at 16 08 34

**Comment by seear** [comment]
cc @kathrynwp

**Comment by budzanowski** [comment]
Yes, we should check that the data is loaded. Corrected.

File: client/my-sites/theme/main.jsx [hunk 2]
```diff
@@ -125,8 +126,19 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
+	getFullLengthScreenshot() {
+		/**
+		 * Location of full length theme screenshot in screenshots array
+		 * depends on themes API v1.2 screenshots field
```
Comment (seear) [inline client/my-sites/theme/main.jsx]: Does this comment add much?

**Comment by seear** [comment]
Code looks good. Tweak the comment if you want, then 🚢

**Pull request merged**